src/components/AvatarWithIndicator.js

```diff
@@ -81,4 +87,7 @@ export default withOnyx({
     cardList: {
         key: ONYXKEYS.CARD_LIST,
     },
+    userWallet: {
```

Comment:
This avatarWithIndicator is going to get so bloated with all these onyx keys 😭

Reply:
😭 Agreed, not sure if there's a better way?